backend/app/db/questions.py

```
@@ -28,17 +118,78 @@ async def create_question(question_data: QuestionCreate, user_id: str) -> Dict[s
+
+        # Convert ObjectIds to strings and prepare return data
+        created_questions = []
+        for i, inserted_id in enumerate(result.inserted_ids):
+            question_dict = question_dicts[i].copy()
+            question_dict["id"] = str(inserted_id)
+            question_dict.pop("_id", None)
+            created_questions.append(question_dict)
+
+        return created_questions
+
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)